lib/storage/src/content_manager/collection_meta_ops.rs

```diff
@@ -245,6 +245,7 @@ pub struct UpdateCollection {
     /// Custom params for Optimizers.  If none - it is left unchanged.
     /// This operation is blocking, it will only proceed once all current optimizations are complete
     #[serde(alias = "optimizer_config")]
+    #[validate(nested)]
     pub optimizers_config: Option<OptimizersConfigDiff>, // TODO: Allow updates for other configuration params as well
```

Comment:
We should likely add another validation rule on `OptimizersConfigDiff`.

Candidate:
- `deleted_threshold`: [0.0, 1.0]